helix-loader/src/lib.rs

```diff
@@ -25,31 +26,83 @@ pub fn initialize_config_file(specified_file: Option<PathBuf>) {
     CONFIG_FILE.set(config_file).ok();
 }
 
-pub fn runtime_dir() -> PathBuf {
+/// A list of runtime directories from highest to lowest priority
+///
+/// The priority is:
+///
+/// 1. `HELIX_RUNTIME` (if environment variable is set)
+/// 2. sibling directory to `CARGO_MANIFEST_DIR` (if environment variable is set)
+/// 3. subdirectory of user config directory (always included)
+/// 4. subdirectory of path to helix executable (always included)
+///
+/// Postcondition: returns at least two paths (they might not exist).
+fn prioritize_runtime_dirs() -> Vec<PathBuf> {
```

Comment:
It seems to me that this could be done in a `Lazy` block and stored as `static RUNTIME_DIRS` so that we only compute this vector once

Reply:
Ah sorry, I missed that this was actually done above!